addons/jest/package.json

```diff
@@ -56,16 +41,14 @@
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
-    "gh-pages": "^1.0.0",
     "jest": "^21.2.1",
     "react": "^16.1.0",
     "react-dom": "^16.1.0",
     "style-loader": "^0.19.0"
   },
   "peerDependencies": {
-    "@storybook/addons": "^3.2.14",
     "prop-types": "^15.6.0",
```

Comment:
It should be a simple dependency, not a peer one

Reply:
I actually meant the `prop-types` package. It's OK for an end user not to have it